src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { getFilteredTodos } from './services/getFilteredTodos';
 
-const USER_ID = 0;
+const USER_ID = 87;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filter, setFilter] = useState<Status>(Status.All);
+
+  const filteredTodos = getFilteredTodos(filter, todos);
+  const activeTodosCount = todos.reduce((acc, cur) => {
+    if (!cur.completed) {
+      return acc + 1;
+    }
+
+    return acc;
+  }, 0);
```

Comment:
In this case you can use filter with todo.completed & lenght property. Not necessary.